contrib/ci-show-dep.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/bash
+source ${DEVENV_WORKSPACE}/scripts/init
```

Comment:
Please correct me if I am wrong: `DEVENV_WORKSPACE` is not in the code base.  Thus it is not available after `devenv use ...` if not run by Github Action, and this line will create directories in the root directory.

I think it's better to exit when the environment variable is not set.

Reply:
Ah, right. Thanks for taking care of a more general case. Will fix.